rama-tcp/src/client/connect.rs

```diff
@@ -23,6 +23,49 @@ use tokio::{
     },
 };
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+enum IPModes {
+    Dual,
+    SingleIpV4,
+    SingleIpV6,
+    DualPreferIpV4
+}
+
+impl Default for IPModes {
```

Comment:
You can easier define this by using `Default` in the `#[derive(..)]` above and set `#[default]` above `Dual,`.